README.md

```diff
@@ -157,6 +157,20 @@ box._ipfs.swarm.connect(pinningNode, () => {
 
 Reference [ipfs-js](https://github.com/ipfs/js-ipfs) for additional options.
 
+### Open a space
+A space is a name spaced section of a users 3Box. Each space has both a public and a private store, and for every space you open the user has grant explicit consent to view that space. This means that if your dapp uses a space that no other dapp use, yours will be the only dapp that is allowed to update the data and read the private store of that particular space. To open a space called `narwhal` you simply call:
```

Comment:
'A space is a name spaced section of a users 3Box' redundant use of `space` is a bit confusing 
`the user has grant explicit` -> `the user has to grant explicit`
`space that no other dapp use` -> `space that no other dapp uses`
`yours will be the only dapp that is allowed to update` -> `only your dapp is allowed to update`

Reply:
Thanks, made some updates!